ui/admin/app/templates/scopes/scope/sessions/index.hbs

```diff
@@ -68,76 +68,64 @@
-              <row.headerCell class='nowrap'>
-                <Rose::Icon
-                  class={{concat
-                    'session-status-'
-                    sessionAggregate.session.status
-                  }}
-                  @name='flight-icons/svg/entry-point-16'
-                  @size='medium'
-                />
-                <Copyable
-                  @text={{sessionAggregate.session.id}}
-                  @buttonText={{t 'actions.copy-to-clipboard'}}
-                  @acknowledgeText={{t 'states.copied'}}
-                >
-                  <code>{{sessionAggregate.session.id}}</code>
-                </Copyable>
-              </row.headerCell>
-              <row.cell>
-                <code>{{sessionAggregate.user.id}}</code>
-                {{#if sessionAggregate.user.accountName}}
-                  <br />
-                  <Hds::Badge @text={{sessionAggregate.user.accountName}} />
-                {{/if}}
-              </row.cell>
-              <row.cell>{{sessionAggregate.target.displayName}}</row.cell>
-              <row.cell>
-                <time
-                  datetime={{format-date-iso
-                    sessionAggregate.session.created_time
-                  }}
+
+      <Hds::Table
+        @model={{@model}}
+        class='sessions-list-table'
+        @columns={{array
+          (hash label=(t 'form.id.label'))
+          (hash label=(t 'resources.user.title'))
+          (hash label=(t 'resources.target.title'))
+          (hash label=(t 'form.started.label'))
+          (hash label=(t 'form.status.label'))
+          (hash label=(t 'titles.actions'))
+        }}
+      >
+        <:body as |B|>
+          <B.Tr>
+            <B.Td>
+              <Rose::Icon
+                class={{concat 'session-status-' B.data.session.status}}
+                @name='flight-icons/svg/entry-point-16'
+                @size='medium'
+              />
+              <Copyable
+                @text={{B.data.session.id}}
+                @buttonText={{t 'actions.copy-to-clipboard'}}
+                @acknowledgeText={{t 'states.copied'}}
+              >
+                <Hds::Text::Code>{{B.data.session.id}}</Hds::Text::Code>
+              </Copyable>
+            </B.Td>
+            <B.Td>
+              <Hds::Text::Code>{{B.data.user.id}}</Hds::Text::Code>
+              {{#if B.data.user.accountName}}
+                <br />
+                <Hds::Badge @text={{B.data.user.accountName}} />
+              {{/if}}
+            </B.Td>
+            <B.Td>{{B.data.target.displayName}}</B.Td>
+            <B.Td>
+              <time datetime={{format-date-iso B.data.session.created_time}}>
+                {{format-date-iso-human B.data.session.created_time}}
+              </time>
+            </B.Td>
+            <B.Td>
+              <SessionStatus @model={{B.data.session}} />
+            </B.Td>
+            <B.Td>
+              {{#if B.data.session.isAvailable}}
```

Comment:
This should check the `can 'cancel session'` ability (which checks if the session is available and if the session has `cancel` authorized action)

Reply:
@lisbet-alvarez is that in reference to the action to the `route-action` to cancel a session?

Reply:
i'm thinking the if statement on line 117 should be replaced with `{{#if (can 'cancel session' B.data.session)}}` so the cancel session button is only displayed when they have cancel action. Or is there a reason we don't want to check cancel permission?

Reply:
@lisbet-alvarez yes good point.  We already check for the availability in the `SessionAbility'